docs/eventing/channels/default-channels-crds.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Default channels"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 2
+type: "docs"
+---
+
+The default channel configuration allows channels to be created without
+specifying an underlying implementation. This leaves the selection of the channel implementation 
+and properties up to the operator. The operator controls the default settings via a
+`ConfigMap`. For example, when `Brokers` or `Sequences` are created, an operator can configure a default channel to use 
+for their underlying channel-based implementations.
+
+Even though this default channel mechanism aims to ease the usability of the system, users can still create their own 
```

Comment:
I think we should describe what a channel does here.  Also, say why someone might want to use a default channel. Right now, we say the default channel "allows channels to be created without specifying an underlying implementation." I'd like us to clarify why someone wouldn't want to specify the implementation to begin with. @nachocano

Reply:
@nachocano  let me know if I should throw some time on your calendar to discuss!

Reply:
Sam, sorry, I've been busy with other stuff. The comment makes sense to me. Let me address it, and I'll get back to you! Thanks!

Reply:
Done, please take another look!